etc/scripts/docenizer.py

```diff
@@ -354,7 +346,7 @@ def docenizer():
     }
 }
 """)
-    print "REMINDER: Check if https://github.com/compiler-explorer/compiler-explorer/issues/2380 is still relevant"
+    print("REMINDER: Check if https://github.com/compiler-explorer/compiler-explorer/issues/2380 is still relevant")
```

Comment:
I guess we should check this issue sometime soon hehe :)

Reply:
Intel PDF's haven't changed since May 2020

Reply:
Oh yuck :'(